### ISSUE TITLE
Homepage information in KISS Linux

### ISSUE THREAD

**Issue by dylanaraps** [repology/repology-updater#946]
Hello,

I just noticed this message on the KISS repology page:

> This repository does not provide homepage information in a way accessible by Repology. This is critical, because homepages are used to resolve project name conflicts, e.g. to distinguinsh similarly named packages of unrelated projects. Bacause of that, this repository is subject for removal in the near future.

The one part I'm particularly worried about is: 

> Bacause of that, this repository is subject for removal in the near future.

KISS doesn't store this information in its package system at all. Each package's homepage is entirely (*and intentionally*) absent from our package system. This is done as the information can still be discerned by the user through a combination of the package's source download URL and its name.

I don't think it's reasonable to request homepage information be added to the KISS package format as it is intentionally omitted by design (*it is not at all useful to the function of the system itself*). 

For the same reasons there are no categories, license information (*the only non-free/non-FOSS package is `linux-firmware` and it may be removed from the repositories entirely*), maintainer information (*`git log` is used*) etc.

Basically, the information in the system solely exists for the *function* of the system. I don't know what to do regarding the above warning. I will not be adding homepages, categories, licenses etc to each package.

I'm happy to hear your thoughts on the issue. :+1:

**Comment by AMDmi3** [comment]
Good catch. I need to change a condition for that alert to check downloads as well as homepage.
KISS is good as download information is available, and Repology already uses that for resolving conflicts.
